### PR TITLE
NETWORK.ETHTOOL: Skip Get/Set msg level test if kernel symbols missing | incomplete

### LINKED ISSUE #341: NETWORK.ETHTOOL-GET-SET-MSG-LEVEL Not supported
Setting the MSG level seems to be unsupported by some kernels and the test fails because of it,
Maybe we could skip the test if the action is not supported?

```
Wed Jun 19 05:28:24 2019 : The network interface is eth0
Wed Jun 19 05:28:24 2019 : Try to set tx_done and rx_status flags on eth0
Wed Jun 19 05:28:24 2019 : Cannot get msglvl: Operation not supported
Wed Jun 19 05:28:24 2019 : Setting the driver message type flags from ethtool is not supported on Linux 4.15.0-1047-azure
```

### PR DESCRIPTION
**Pull request by stefanlupsa**
Fixes #341 
Check for get / set msg level symbols in the lib of the current kernel, if missing skip the test.

### CODE REVIEW

**Comment by stefanlupsa** [comment]
Canonical : UbuntuServer : 16.04-LTS : 16.04.201906170 was known to fail the test for this reason:
```
[azure] [LISAv2 Test Results Summary]
[azure] Test Run On           : 06/24/2019 14:09:26
[azure] ARM Image Under Test  : Canonical : UbuntuServer : 16.04-LTS : 16.04.201906170
[azure] Total Test Cases      : 1 (0 Passed, 0 Failed, 0 Aborted, 1 Skipped)
[azure] Total Time (dd:hh:mm) : 0:0:2
[azure] 
[azure]    ID TestArea             TestCaseName                                                                TestResult TestDuration(in minutes) 
[azure] -------------------------------------------------------------------------------------------------------------------------------------------
[azure]     1 NETWORK              ETHTOOL-GET-SET-MSG-LEVEL                                                      SKIPPED                 0.68 
```

**Comment by LiliDeng** [comment]
@stefanlupsa Please try against this image RedHat RHEL 7-RAW 7.6.2019062020, it is compression format, seems end with xz

**Comment by stefanlupsa** [comment]
> @stefanlupsa Please try against this image RedHat RHEL 7-RAW 7.6.2019062020, it is compression format, seems end with xz

@LiliDeng  I'm already testing against RHEL daily image for this issue, will post results when finished, can you clarify what you mean with the compression format please?

File: Testscripts/Linux/ETHTOOL-MSG-LEVEL.sh [hunk 1]
```diff
@@ -23,6 +23,31 @@ UtilsInit
...
+
+kernel_version=$(uname -r)
+# if the module is not builtin
+if [ $(grep -c netvsc < /lib/modules/"$kernel_version"/modules.builtin) == 0 ]; then
+    # get the path to the netvsc kernel module
+    kernel_module=$(ls /lib/modules/"$kernel_version"/kernel/drivers/net/hyperv/hv_netvsc.ko*)
+    # if the module is archived as xz, extract it to check symbols
+    if [ $(echo "$kernel_module" | grep -c ".xz") -ne 0 ]; then
+        cp "$kernel_module" .
+        xz -d $(basenam $kernel_module)
```
Comment (LiliDeng) [inline Testscripts/Linux/ETHTOOL-MSG-LEVEL.sh:40]: basename?

File: Testscripts/Linux/ETHTOOL-MSG-LEVEL.sh [hunk 2]
```diff
@@ -23,6 +23,31 @@ UtilsInit
 #######################################################################
 # Main script body
 #######################################################################
+# Check if feature is suppoorted by the kernel
```
Comment (LiliDeng) [inline Testscripts/Linux/ETHTOOL-MSG-LEVEL.sh:26]: supported

**Review by LiliDeng — CHANGES_REQUESTED** [review]

**Comment by chvalean** [comment]
Talking internally with Haiyang on how to handle this check if the netvsc module is built-in.
Will re-open the PR with the changes once clarified.

**Pull request closed without merging**